src/components/dataset-recommendations/connector.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux'
+import DatasetRecommendations from './dataset-recommendations'
+import { getDatasetRecommendations, getDataSet } from '../../store/selectors'
+import { datasetRecommendations } from '../../store/actions'
+
+const mapStateToProps = state => {
+    return {
+        recommendations: getDatasetRecommendations(state),
+    }
+}
+
+const mapDispatchToProps = dispatch => ({
+    getRecommendations: datasetId => dispatch(datasetRecommendations(datasetId))
```

Comment:
It seems really weird that those recommendations are consumed by other components than this one, but kind of rely on it being mounted to get them. What do you think of:
A) having both forms of the recommendation view trigger a recommendation load (they would both have connectors)
B) having the `dataset-view` do the recommendation load and have the components not be connected

Reply:
This bothered us and then we forgot about it. I think that we should be getting retrieving the recommendations in dataset-view and passing it down from there.